packaging/installer/kickstart-static64.sh

```diff
@@ -207,13 +207,15 @@ safe_sha256sum() {
 }
 
 mark_install_type() {
-    install_type_file="/opt/netdata/etc/netdata/.install-type"
+  install_type_file="/opt/netdata/etc/netdata/.install-type"
+  if [ -e "${install_type_file}" ]; then
```

Comment:
This should be `-f` to make sure it’s actually a file. Otherwise we’re just checking that the path exists (it could be a directory or even a socket for all we know).

Reply:
Thanks, fixed.